### PR TITLE
Fix for use this components in Android

### PR DESCRIPTION
**Pull request by ethaizone**
In android,  StatusBarIOS will become null so this pull requests will protect this error and make this component can run fine on android.

### CODE REVIEW

**Comment by ethaizone** [comment]
screen shot 2558-09-19 at 6 58 23 pm

File: index.js [hunk 1]
```diff
@@ -174,9 +174,15 @@ const NavigationBar = React.createClass({
 
   render() {
     if (this.props.statusBar === 'lightContent') {
-      StatusBarIOS.setStyle('light-content', false);
+      if (StatusBarIOS)
```
Comment (chirag04) [inline index.js]: can you keep the indentation consistent?

File: index.js [hunk 2]
```diff
@@ -174,9 +174,15 @@ const NavigationBar = React.createClass({
...
+      {
```
Comment (chirag04) [inline index.js]: this should on same line as if

File: index.js [hunk 3]
```diff
@@ -174,9 +174,15 @@ const NavigationBar = React.createClass({
...
+        StatusBarIOS.setStyle('light-content', false);
+      }
     } else if (this.props.statusBar === 'default') {
-      StatusBarIOS.setStyle('default', false);
+      if (StatusBarIOS)
```
Comment (chirag04) [inline index.js]: indentation

File: index.js [hunk 4]
```diff
@@ -174,9 +174,15 @@ const NavigationBar = React.createClass({
...
+      {
```
Comment (chirag04) [inline index.js]: keep it on same line as if

**Comment by ethaizone** [comment]
Thank @chirag04. I update indent that you mention. It's come from I write Laravel4. LOL

**Comment by Kureev** [comment]
Hm, do Android have analog for `light-content` / `default`?

**Comment by ethaizone** [comment]
I don't know. I'm web app developer so I cannot answer this question. /sob

**Comment by Kureev** [comment]
After a while, I decided that navbar component for Android should be done in a different way, have different behavior, appearance and interactions, so I propose not to use this component for Android.

**Pull request closed without merging**